### PR TITLE
Fixed eslint issues, added favicon.ico file

### PR DESCRIPTION
**Pull request by clayboyd**
eslint issues:  missing semi-colons, missing next() callback on middleware components

changed req.param('name') to req.body.name.. param() method is deprecated.

Added missing favicon.ico, caused a runtime error indicating a missing module named 'ico'

### CODE REVIEW

**Comment by shaunwarman** [comment]
👍

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 node_modules
 .build
 .idea
-
+npm-debug.log
```
Comment (grawk) [inline .gitignore]: can you change to `*.log` to cover future use cases as well please?

**Review by grawk — CHANGES_REQUESTED** [review]
just a minor point on the `.gitignore` file. Otherwise looks good.

**Comment by clayboyd** [comment]
Fixed changes to .gitignore...  npm-debug.log ==> *.log

**Comment by grawk** [comment]
@clayboyd did you close this PR erroneously?

**Comment by clayboyd** [comment]
whoops... yep, I did...

**Review by grawk — APPROVED** [review]

**Comment by grawk** [comment]
Thanks!

**Pull request merged**